### PR TITLE
Update to openssl 1.1.1f

### PR DESCRIPTION
**Pull request by kumpelj**
script fails as it can not download old version

*Issue #, if available:*

*Description of changes:*
Update 1.1.1d to 1.1.1f

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: kinesis-video-native-build/install-script [hunk 1]
```diff
@@ -103,14 +103,14 @@ if [[ $build_producer == TRUE ]]; then
   echo "Checking openssl at $DOWNLOADS/local/lib/libssl.a"
   if [ ! -f $DOWNLOADS/local/lib/libssl.a ]; then
     echo "openssl lib not found. Installing"
-    if [ ! -f $DOWNLOADS/openssl-1.1.1d.tar.gz ]; then
+    if [ ! -f $DOWNLOADS/openssl-1.1.1f.tar.gz ]; then
       cd $DOWNLOADS
       #download_cmd="curl -L "https://www.openssl.org/source/openssl-1.1.1d.tar.gz" -o "openssl-1.1.1d.tar.gz" "
```
Comment (disa6302) [inline kinesis-video-native-build/install-script:108]: Thank you for your contribution @kumpelj ! One small change and you are good to go. 

Can you change the comment to reflect the version?

**Comment by disa6302** [comment]
@kumpelj , 

Can you also explain what failed while running the install-script with older openssl version? Thanks!

**Comment by kumpelj** [comment]
> @kumpelj ,
> 
> Can you also explain what failed while running the install-script with older openssl version? Thanks!

as the old version does not exists any longer the curl command fails and install script aborted

**Comment by disa6302** [comment]
Sorry for the delay @kumpelj. Merging it now.

**Pull request merged**